src/Paramore.Brighter/Transforms/Transformers/CloudEventsTransformer.cs

```diff
@@ -84,9 +97,160 @@ public Message Wrap(Message message, Publication publication)
         message.Header.Subject = _subject ?? publication.Subject;
         message.Header.ContentType = _dataContentType ?? publication.ContentType;
         message.Header.SpecVersion = _specVersion ?? message.Header.SpecVersion;
+
+        if (_format == CloudEventFormat.Binary)
+        {
+            return message;
+        }
+
+        var cloudEvent = new CloudEventMessage
+        {
+            Id = message.Id,
+            SpecVersion = message.Header.SpecVersion,
+            Source = message.Header.Source,
+            Type = message.Header.Type,
+            DataContentType = message.Header.ContentType,
+            DataSchema = message.Header.DataSchema,
+            Subject = message.Header.Subject,
+            Time = message.Header.TimeStamp,
+            Data = JsonSerializer.Deserialize<JsonElement>(message.Body.Value)
```

Comment:
The deserialize here could throw. We shoud catch the exception, log, and return the untransformed original message